apps/www/_blog/2022-09-08-choosing-a-postgres-primary-key.mdx

```diff
@@ -59,18 +59,15 @@ SELECT COUNT(*) + 1, 'new@example.com', 'new_user' FROM users;
 
 After running that query, we can double check our results:
 
-```sql
 SELECT * FROM users;
-```
 
 | id | email            | name     |
 |----|------------------|----------|
 | 1  | new@example.com  | new user |
 | 2  | new2@example.com | new user |
```

Comment:
I think these values should be in code. EG:

| id | email            | name     |
|----|------------------|----------|
| `1`  | `new@example.com`  | `new user` |
| `2`  | `new2@example.com` | `new user` |